apps/site/src/content/blog/astrolicious-evolved.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Astrolicious: Evolved"
+draft: false
+publishDate: 2024-09-18
+authors:
+  - name: Alexander Niebuhr
+    url: https://github.com/alexanderniebuhr
+  - name: Florian Lefebvre
+    url: https://github.com/florian-lefebvre
+---
+
+## We are growing 🌱 🌿 🌳
+
+Astrolicious is growing daily, we have gathered over 100 members on our [Discord](https://chat.astrolicious.dev) server and we are excited to see the enthusiasm inside our community. People are sharing their ideas, asking questions and helping each other with their own blog posts, tutorials or examples.
+
+Additionally we have continued our efforts to get more community members involved with decision making, so we are proud to announce that [Fryuni (Luiz Ferraz)](https://github.com/fryuni), has joined the team to shape our future.
+
+## Sunsetting Maintenace Efforts 🌇
+
+Despite the momentum, we have struggled with our dream to offer shared maintenance to our projects. We can't commit to provide maintenance in an amount, we think is necessery for projects to succeed and grow.
+
+We have made a tough decision. We are sunsetting our shared code maintance efforts with immediate effect. That means there are not any Astrolicious maintained code project anymore. But our current [projects](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
```

Comment:
```suggestion
We have made a tough decision. We are sunsetting our shared code maintenance efforts, effective immediately. This means there are no longer any Astrolicious maintained community code projects. But our own [project initiatives](/projects) will stay part of Astrolicious and will continue to be maintained by their original authors and engaged community members.
```
Not exactly sure how you want to refer to the things that are gone and the things that stay, but that could be one way. I'm assuming you mean "the projects we initiated" are staying and "projects who came to find a home here" are the ones that have left.